governance-exercises/Poll Exercise 03.md

```diff
@@ -0,0 +1,110 @@
+
+The structure/content of the poll also fails on a few fronts. There's no introduction to, or summary of, the essential context that participants would need prior to meaningful participation in the poll. There's no Abstain option, which can be a valuable signal for voters who wish to express ambivalence or proactively protest. In each poll option, two different values/scenarios are combined, making any "signal" impossible to discern. The fourth poll option of ''Something Else" is similarly useless. Finally, the title of Emilia's post, "Standing Order Change", is likely to cause significant misunderstanding: voters may believe that the poll (on the logic of the future ALM order) represents the "standing order change" referenced in the title. 
+
+The biggest problem: Emilia's post may lead the community to believe that the poll outcomes have a determinative say in the logic of the replacement ALM order. If Emilia wishes to gauge community sentiment and is yet aware of her mandate as an Arranger, then she would have to acknowledge that deciding the logic of the ALM transfer orders ultimately lies within the purview of domain experts. 
+
+### Re: Faraz's Reply
+
+Faraz's reply doesn't help matters. There is his use of the singular "this", in conjunction with the plural "changes". Is this a typo? It's not clear what he's approving, either. There are two Standing Orders at issue. The first is the proposed Order relating to revenue return, which has obvious ambiguities. The second is the current Order relating to ALM transfers which Emilia improperly "invalidated" in her Forum post. It is possible that Faraz, in issuing such a confusing reply and approving either/both Order(s), has not fulfilled his mandate as a Facilitator. 
+
+Finally: to authorize an item for a poll, the Facilitator and advisory teams must explicitly confirm the proposed change is reasonable and suitable for implementation. Faraz's comment about the ALM structure adds to the mixed signals. This would seem to indicate that he is *not* authorizing a poll for the ALM-related order, because the advisory teams' evaluation on that item is forthcoming. 
+
+---
+
+### Forum Response, in Brief  
+
+To @Emilia:
+
+- I would post a public reply on Emilia's Forum thread.
+- I would explain why a standard governance poll cannot be run in this instance and give her the above-mentioned timeline for future reference (all needed information by Thursday, etc.). 
+- I would not mention the non-standard poll or the MIP24 emergency vote unless Emilia brought them up.
+- I would point out, at a high level only, the shortfalls in her Post and Forum Poll. (I am aware that Faraz has the more direct working relationship and scope of authority/review.) I would emphasize my mandate as a Governance Facilitator in explaining the shortfalls.  
```

Comment:
I would avoid leaning too hard on the 'my mandate is forcing me to tell you you're wrong' thread, as a general comment. Obviously hard to see how you'd handle it in specifics given this is a summary, you're most likely fine. 

Basically, if you are constantly saying: "I'm governance facilitator, and its my job to be mean" I don't think it comes off particularly well. If someone comes back and is all 'who do you think you are' then fine.